.gitignore

```diff
@@ -348,3 +348,42 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+appsettings.Development.json
+appsettings.json
+Properties/launchSettings.json
+wwwroot/favicon.ico
```

Comment:
we need everything inside wwwroot. so you can remove wwwroot from gitingore and then also add those to your commit